stylesheet.css

```diff
@@ -1,195 +1,1040 @@
-html {     
-  box-sizing: border-box; 
-} 
-
 *, *:before, *:after {     
-  box-sizing: inherit; 
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
 }
 
-body {
-  background-color: white;
-  font-family: Verdana, Arial, sans-serif;
-  margin-left: 25%;
-  margin-right: 25%;
+ul {
+  list-style-type: none;
 }
 
 li {
-  display: inline; 
+  text-decoration: none;
 }
-
 /*-- link --*/
+a {
+  text-decoration: none;
+  color: initial;
+}
+
 a:link {
   text-decoration: none;
   color: black;
 }
 
-/*-- link hovered --*/
-a:hover {
-    background-color: blue;
-    color: white;
+/*body*/
+body {
+  font-family: 'Roboto', sans-serif;
+  font-size: 1.25rem;
+  width: 100%;
+  margin: auto;
+  background-position: 0 5rem;
+  background: url("images/header.png") no-repeat;
+  background-position-y: 5rem;
+  box-sizing: border-box;
+  line-height: 1;
+}
+
+body section {
+  padding: 0 1.25rem;
 }
 
-/*-- sections header --*/
-.section-header {
+body > header {
+  padding: 0 calc((100vw - 52rem)/2);
+  width: 100%;
+  background-color: white;
+  margin: auto;
+  height: 5rem;
+  position: fixed;
+  display: flex;
+  align-items: center;
+  font-family: "Montserrat";
+  z-index: 1;
+}
+
+#menu-company-logo {
+  background-image: url("images/rebased-logo-full.png");
+  text-indent: -9999px;
+  width: 10rem;
+  height: 3rem;
+  background-repeat: no-repeat;
+  background-size: contain;
+  background-position: center;
+  background-color: white;
+  margin-left: 1rem;
+}
+
+body #main-nav:before {
+  content: '\2630';
+  position: absolute;
+  top: 0;
+  right: 0;
+  font-size: 2rem;
+  padding: 1.2rem 1.6rem;
+  height: 5rem;
+}
+
+body header #main-nav ul {
+  display: none;
+}
+
+body.menu-mobile-active {
+  overflow-y: hidden;
+  transition: 5s ease;
+  -webkit-transition: 5s ease;
+}
+
+body.menu-mobile-active #main-nav:before {
```

Comment:
If in media queries you're specyfing this also with `header` selector, you should add it here too.